frontend/src/admin/components/sidebar/dropdown/dropdownMenu.scss

```diff
@@ -0,0 +1,43 @@
+@import "src/styles/main";
+
+.c-menu {
+  position: relative;
+
+  &__button{
+    &--active {
+      .c-dropdown {
+        opacity: 1;
+        visibility: visible;
+        position: absolute;
+        background-color: $card-color;
+        border-radius: 5px;
+        top: 60px;
+        left: 0;
+        min-width: 150px;
+        box-shadow: 0 1px 8px rgba(0, 0, 0, 0.3);
+        transform: translateY(-20px);
+        transition: opacity 0.4s ease, transform 0.4s ease, visibility 0.4s;
+
+        p {
+          color: $contrast-color;
+          height: 30px;
+          display: flex;
+          flex-direction: column;
+          justify-content: center;
+          align-items: flex-start;
+          padding: 1rem;
+          border-bottom: 1px solid $border-color;
+
+          &:last-child {
+            border-bottom: none;
+          }
+
+          &:hover {
+            font-weight: bold;
+            transition:  0.4s ease;
+          }
+        }
```

Comment:
This is styling of the `DropdownMenuItem` and not the `DropdownMenu`. Should be separated.

Reply:
✅